lib/Onyx.js

```diff
@@ -722,13 +718,25 @@ function init({
     ])
         .then(deferredInitTask.resolve);
 
-    // Update any key whose value changes in storage
-    registerStorageEventListener((key, newValue) => {
-        cache.set(key, newValue);
-        keyChanged(key, newValue);
-    });
+    if (synchronizeInstances) {
+        Storage.synchronizeInstances({keyChanged});
+    }
```

Comment:
Relaying updates to other tabs happens in a similar manner as before, but since it's tied to IndexedDB (and localforage) usage it's better to manage it in Onyx